aqua/phantomjs-qt/Portfile

```diff
@@ -91,7 +91,7 @@ array set modules {
         {"Qt WebKit" "Qt WebKit Widgets"}
         ""
         "variant overrides: ~docs ~examples"
-        "revision 1"
+        "revision 2"
```

Comment:
Revision incremented because this changes runtime behavior even on earlier macOS. If you do not want this change affecting/rebuilding on older macOS, please ask me to make this change specific to macOS 11+.

Reply:
Seems reasonable to rev-bump, given the circumstances. I'm fine with it!